packages/perseus/src/util/colors.ts

```diff
@@ -81,13 +83,9 @@ const KhanColors = {
     GRAY_I: "#21242C",
     KA_BLUE: "#314453",
     KA_GREEN: "#71B307",
-    // Don't actually use _BACKGROUND! Make things transparent instead. The
-    // background color used in exercises is subject to change at the whim
-    // of any redesigns.
-    _BACKGROUND: "#FDFDFD", // TODO(eater): Get rid of this altogether.
```

Comment:
Got rid of this part since both comments seem to think it is not needed. Replaced where they are used with just `"transparent"`.